anchor/common/qbft/src/lib.rs

```diff
@@ -20,156 +20,98 @@ mod validation;
 #[cfg(test)]
 mod tests;
 
-type RoundChangeMap<D> = HashMap<OperatorId, Option<ConsensusData<ValidatedData<D>>>>;
+type RoundChangeMap<D> = HashMap<OperatorId, Option<ConsensusData<D>>>;
+
+pub trait Data: Debug + Clone {
```

Comment:
```rust
pub trait Hashable: Debug + Clone {
    type Digest: Debug + Clone + Eq + Hash;
    
    fn digest(&self) -> Self::Digest;
}
```

What do you think about it? The motivation is to make the trait name more descriptive and avoid using `Hash` twice - it seems like a recursive definition.